admin/RELEASE.md

```diff
@@ -40,25 +40,32 @@ Copy the resulting content into **CHANGELOG.md** and adjust the format to match
 * Create **user_guide_src/source/installation/upgrade_{ver}.rst**, fill in the "All Changes" section, and add it to **upgrading.rst**
 * Commit the changes with "Prep for 4.x.x release" and push to origin
 * Create a new PR from `release-4.x.x` to `develop`:
-	* Title: "Prep for 4.x.x release"
-	* Decription: "Updates changelog and version references for `4.x.x`." (plus checklist)
+    * Title: "Prep for 4.x.x release"
+    * Decription: "Updates changelog and version references for `4.x.x`." (plus checklist)
```

Comment:
I can't find supporting evidence but I thought sub-bullets were supposed to be dashed? Maybe that's something other than Markdown?
```suggestion
    - Title: "Prep for 4.x.x release"
    - Decription: "Updates changelog and version references for `4.x.x`." (plus checklist)
```

Reply:
> I thought sub-bullets were supposed to be dashed?

I can't find such a rule.
https://github.github.com/gfm/#example-272

> A [bullet list marker](https://github.github.com/gfm/#bullet-list-marker) is a -, +, or * character.
https://github.github.com/gfm/#bullet-list-marker